fiksi/src/elements/mod.rs

```diff
@@ -110,17 +110,36 @@ impl<'el> sealed::ElementInner for Line<'el> {
 }
 
 /// A circle defined by a centerpoint and a radius.
-///
-/// TODO: use.
 #[derive(Debug)]
-pub struct Circle {
+pub struct Circle<'el> {
     /// The center of the circle.
-    pub center: ElementHandle<Point>,
+    pub midpoint: &'el ElementHandle<Point>,
```

Comment:
Should this comment change to say "center (or midpoint)" or similar? I'm not sure why midpoint is used here though.